en_ewt-ud-dev.conllu

```diff
@@ -20015,7 +20015,7 @@
 # sent_id = answers-20111107180248AAnQ3aE_ans-0002
 # newpar id = answers-20111107180248AAnQ3aE_ans-p0002
 # text = Ya can't.
-1	Ya	ya	PRON	PRP	_	2	nsubj	2:nsubj	_
+1	Ya	you	PRON	PRP	Case=Acc|Person=2|PronType=Prs|Style=Coll	2	nsubj	2:nsubj	CorrectForm=you
```

Comment:
Do we want to specify a non-colloquial CorrectForm and lemma for colloquial variants? I sort of feel like if we are saying they project a certain Style then they are different lexical entries. And it would feel weird to normalize "y'all" (plural) to "you".

Reply:
I'd be for it, and wanted to add that this is already the case in GUM for the following irregular forms, including lower-case i:

  * Form=em; Lemma=they; CorrectForm=them
  * Form=i; Lemma=I; CorrectForm=I
  * Form=u; Lemma=you; CorrectForm=you
  * Form=ya; Lemma=you; CorrectForm=you

Reply:
GUM lists "ya" as a typo, which I think is wrong.

I was arguing against treating this as a correction, and proposing instead that they are different lexical entries (sorry if my phrasing was unclear). Would GUM normalize "y'all" to "you"?

Reply:
Well, I certainly agree that "ya" isn't a typo as I understand the English word typo; but natively, GUM has no typo annotation whatsoever. The native GUM target hypothesis annotations just give a 'standard' English rendition of anything that is non-standard, which includes both genuine typos and other features of non-standard English spelling or speech. 

I think we've discussed this in some other issue, but currently populating the `Typo` and `CorrectForm` annotations in the CoNLL-U version of GUM happens automatically, so a lot of different things get called those names which probably should have different names, but I don't think this will be resolved for the upcoming UD release.

Reply:
OK. Since the Style approach is more precise let's go with that for now for EWT (and maybe GUM will adopt later). It doesn't affect that many tokens, but @rhdunn has implemented a nice set of checks.

Reply:
I've reverted the CorrectForm for those entries in the current commits.